fishtest/fishtest/rundb.py

```diff
@@ -315,10 +324,15 @@ def flush_buffers(self):
         self.run_cache_lock.release()
 
     def scavenge(self, run):
-        old = datetime.utcnow() - timedelta(minutes=110)
+        # print("scavenge ", run["_id"])
+        dead_task = False
+        old = datetime.utcnow() - timedelta(minutes=3)
```

Comment:
@ppigazzini Adjust timeout here: `minutes=30`

I expect that we remove `utils/scavenge.py` from the crontab, otherwise we need to change it there too

Reply:
@tomtor thanks for the info, I will comment out `scavenge.py` on PROD crontab